content/docs/tasks/telemetry/access-log/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Envoy's Access Logging
+description: This task shows you how to configure Envoy proxies to print access log to their standard output.
+weight: 9
+keywords: [telemetry]
+---
+
+The simplest kind of Istio logging is
+[Envoy's access logging](https://www.envoyproxy.io/docs/envoy/latest/configuration/access_log).
+Envoy proxies print access information to their standard output.
+The standard output of Envoy's containers can then be printed by the `kubectl logs` command.
+
+{{< boilerplate before-you-begin-egress >}}
+
+{{< boilerplate start-httpbin-service >}}
+
+## Enable Envoy's access logging
+
+Edit the `istio` configuration map:
+
+{{< text bash >}}
+$ kubectl edit cm istio -n istio-system
+{{< /text >}}
+
+Set the value of `global.proxy.accessLogFile` to "/dev/stdout".
+Mind escaping quotation marks with backward slashes (`\"`).
```

Comment:
Mind escaping -> Be sure to escape

Reply:
Fixed in https://github.com/istio/istio.io/pull/3531/commits/75b2ac3f57f02489da2f81039ff6679dbca1f97d